Aztec/Classes/TextKit/LayoutManager.swift

```diff
@@ -8,8 +8,17 @@ import QuartzCore
 class LayoutManager: NSLayoutManager {
 
     /// Blockquote's Left Border Color
-    ///
-    var blockquoteBorderColor: UIColor? = UIColor(red: 0.52, green: 0.65, blue: 0.73, alpha: 1.0)
+    /// Include 3 nested values
+    
+    var blockquoteBorderColor = [0 : UIColor(hexString: "9252FF")!,
+                                1 : UIColor(hexString: "5A4AE8")!,
+                                2 : UIColor(hexString: "5D77FE")!,
+                                3 : UIColor(hexString: "4A8EE8")!,
+                                4 : UIColor(hexString: "52CAFF")!]
+    
+    /// Update this accordingly to the highest index value in the dictionary above
+    var quoteMaxDepth: Int = 4
```

Comment:
This could be implemented like this:
``` var quoteMaxDepth = { get { return blockquoteBorderColor.count } }
in order to autosync with the above array/dictionary

Reply:
oh right thats better, only thing it would have to be `count-1` because the way I use this var is to point to a particular index so 0-4 rather than 1-5

Reply:
ok with the array this var really isn't necessary anymore; will open the new PR